packages/components/pie-textarea/test/visual/pie-textarea.spec.ts

```diff
@@ -175,3 +175,93 @@ test.describe('Label and Character count:', () => {
         await percySnapshot(page, 'Textarea RTL - with label and character count', percyWidths);
     });
 });
+
+test.describe('Assistive text and statuses:', () => {
```

Comment:
I think we can reduce the amount of screenshots by combining all the variations into RTL and LTR. We do that for other components like button and checkbox etc